test/visual-diff/visual-tests

```diff
@@ -181,6 +181,16 @@
       "name": "Video rotate-to-fullscreen",
       "loading_complete_css": ["video.i-amphtml-replaced-content"]
     },
+    {
+      "url": "examples/visual-tests/amp-sticky-ad/amp-sticky-ad.amp.html",
+      "name": "AMP Sticky Ad",
+      "viewport": {
+        "width": 400,
+        "height": 600
+      },
+      "loading_complete_css": [".i-amphtml-sticky-ad-layout"],
```

Comment:
You may want to wait for `amp-sticky-ad[visible]`. That's applied after ad has rendered compared to `.i-amphtml-sticky-ad-layout` is applied right after sticky-ad layout.

Reply:
Done